networkx/algorithms/__init__.py

```diff
@@ -1,3 +1,4 @@
+from networkx.algorithms.applications import *
```

Comment:
Maybe this would be better in a `networkx.algorithms.tsp` module; all the other algorithms modules are named for the type of problem they solve (clique, independent set, etc.).